Create factura_cliente.php

c-next opened the pull request:
Modifica la forma en que se asignan los números de factura.
En Colombia se maneja una numeración consecutiva, independiente al ejercicio.
Permite agregar los prefijos de facturación asignados por la DIAN, en Contabilidad -> Series.

c-next committed:
Create factura_cliente.php

Modifica la forma en que se asignan los números de factura.
En Colombia se maneja una numeración consecutiva, independiente al ejercicio.
Permite agregar los prefijos de facturación asignados por la DIAN, en Contabilidad -> Series.

NeoRazorX commented on model/factura_cliente.php:
```diff
@@ -0,0 +1,95 @@
+         $this->numero = $num;
+      }
+      ///aca se define la cantidad de ceros antes del número consecutivo en %06s
+         $this->codigo = $this->codserie.sprintf('%06s', $this->numero);
```
Código debe ser único. Con esta implementación estás comprobando por serie, pero la serie no se incluye en el código, así que si creo otra serie, volverá a empezar en 1 generando duplicados, es decir:

La primera factura de la serie A será 000001. Ya la primera factura de la serie B será 000001, produciendo un error de código duplicado.

Solución: no tener en cuanta la serie. Quitarla del where de la línea 47.

NeoRazorX reviewed (changes requested):
Se necesita resolver el problema de códigos duplicados.

c-next committed:
Update factura_cliente.php

Se corrige el error de código de factura repetido, agregándole la serie al iniciar el código.

De este modo para la serie A el código seria A000001 y para la serie B el código seria B000001 y así, dependiendo dela serie.
